roles/traefik/templates/traefik.toml.j2

```diff
@@ -1,6 +1,6 @@
-port = ":80"
 graceTimeOut = 10
 logLevel = "ERROR"
+defaultEntryPoints = ["http", "https"]
```

Comment:
[This line appears twice](https://github.com/CiscoCloud/mantl/pull/1378/files#diff-f0aec6dec4496d4f0d42e882b8f1cc71R17), is that intended?